apps/antalmanac/src/components/Calendar/calendar.css

```diff
@@ -40,6 +40,26 @@
     font-size: 0.9rem;
 }
 
+.dark-mode .rbc-time-content {
+    border-top-color: rgb(58, 62, 65);
+}
+
+.dark-mode .rbc-time-content > * + * > * {
+    border-left-color: rgb(58, 62, 65);
+}
+
+.dark-mode .rbc-time-view {
+    border-color: rgb(58, 62, 65);
+}
+
+.dark-mode .rbc-time-header-content {
+    border-left-color: rgb(58, 62, 65);
+}
+
+.dark-mode .rbc-time-header.rbc-overflowing {
+    border-right-color: rgb(58, 62, 65);
+}
+
```

Comment:
If possible, you can see if these CSS styles can be embedded in a `styles` prop, so that way you can refer to the theme if possible. For example:
```tsx
function Calendar() {
  const theme = useTheme()

  return (
    <div style={{ '.dark-mode .rbc-time-view': { backgroundColor: theme.palette.main }}}>
    </div>
  )
}
```

Reply:
Hi Brian, I didn't figure out how to directly apply CSS selectors within the style attribute.
Can you help me out?